include/PetriEngine/Colored/Reduction/ColoredReducer.h

```diff
@@ -76,21 +84,33 @@ namespace PetriEngine::Colored {
                 return _builder.inhibitors();
             }
 
-            CArcIter getInArc(uint32_t pid, const Colored::Transition &tran) const;
+            PetriEngine::Colored::Transition &alterableTransitionReference(uint32_t id) const {
+                return _builder._transitions[id];
+            }
```

Comment:
Den her gør ondt helt ind i mit hjerte designmæssigt. Jeg kan se, at det eneste du bruger den til, er at sætte `newTran.skipped = false` og `newTran.inhibited = false`. Disse to ændringer bør gøres i `newTransition()` funktionen anyway.

Reply:
Det er den sidste remnant af hvordan den første implementering gjorde alt.